app/controllers/miq_ae_class_controller.rb

```diff
@@ -2014,8 +2013,8 @@ def fields_get_form_vars
         return
       end
       new_fields = {}
-      field_attributes.each_with_object({}) { |field| field = field.to_sym
-      new_fields[field] = @edit[:new_field][field]}
+      field_attributes.each_with_object({}) { |field| field
+      new_fields[field] = @edit[:new_field][field.to_sym]}
```

Comment:
@h-kataria 
can we clean that up. Also the |field| field doesn't make sense, I think earlier it was field = field.to_sym and then the field.to_sym got deleted leaving a dangling field